packages/aws-cdk-lib/aws-logs/lib/log-group.ts

```diff
@@ -84,6 +84,28 @@ export interface ILogGroup extends iam.IResourceWithPolicy {
    * Public method to get the physical name of this log group
    */
   logGroupPhysicalName(): string;
+
+  // /**
+  //  * Return the given named metric for this Log Group
+  //  *
+  //  * @param metricName The name of the metric
+  //  * @param props Properties for the metric
+  //  */
+  // metric(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the number of logs ingested
```

Comment:
Maybe use the description from https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch-Logs-Monitoring-CloudWatch-Metrics.html